source/db/migrate/20150325112806_create_games.rb

```diff
@@ -0,0 +1,14 @@
+class CreateGames < ActiveRecord::Migration
+  def change
+    create_table :games do |t|
+      t.references :user1, class_name: "User"
```

Comment:
you really only need a winner / looser table columns here, having both users and the winner / losers is a repeat of data :cry: :panda_face: